pkg/api/v1beta1/types.go

```diff
@@ -726,10 +844,12 @@ type NodeResources struct {
 type ResourceName string
 
 const (
-	// CPU, in cores. (floating point w/ 3 decimal places)
+	// CPU, in cores. (500m = .5 cores)
 	ResourceCPU ResourceName = "cpu"
-	// Memory, in bytes.
+	// Memory, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
 	ResourceMemory ResourceName = "memory"
+	// Volume size, in bytes (e,g. 5Gi = 5GiB = 5 * 1024 * 1024 * 1024)
+	ResourceStorage ResourceName = "storage"
```

Comment:
As discussed elsewhere, I'd anticipate storage space (bytes), storage IOPs and perhaps storage bandwidth (bytes/sec) here, so "storage" is not a good name here. I'd go for something logically along the lines of "storage.space" or "storage.size".